modules/dds/texture_loader_dds.cpp

```diff
@@ -404,8 +404,13 @@ Ref<Resource> ResourceFormatDDS::load(const String &p_path, const String &p_orig
 		// BC compressed.
 		uint32_t size = MAX(info.divisor, w) / info.divisor * MAX(info.divisor, h) / info.divisor * info.block_size;
 
-		ERR_FAIL_COND_V(size != pitch, Ref<Resource>());
-		ERR_FAIL_COND_V(!(flags & DDSD_LINEARSIZE), Ref<Resource>());
+		if (flags & DDSD_LINEARSIZE) {
+			// Image exporter specified a size for the top-level image; it should match our calculation
+			ERR_FAIL_COND_V(size != pitch, Ref<Resource>());
```

Comment:
As said in the issue this needs to be contextualised so it's clear what's wrong, as it's not clear from this message that it depends on the flag

Reply:
Sorry, I guess it wasn't clear to me what you meant by that comment. I wasn't previously aware of the existence of `ERR_FAIL_COND_V_MSG()`, which is what I'm assuming you want me to do?

Reply:
In this case yeah, sorry for unclear, tried to make the same point in there 🙂

Reply:
The wording sounds good 🙂